applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -42,6 +42,8 @@ LOG_MODULE_REGISTER(cis_headset, CONFIG_BLE_LOG_LEVEL);
 #define NET_BUF_POOL_PTR_ITERATE(i, ...) IDENTITY(&iso_tx_pool_##i)
 LISTIFY(CONFIG_BT_ASCS_ASE_SRC_COUNT, NET_BUF_POOL_ITERATE, (;))
 
+static atomic_t iso_tx_pool_alloc;
```

Comment:
Unsure if this should be an atomic or a counting semaphore

Reply:
Since we are not going to wait (like using k_sem_take/k_sem_give for controlling the process) but only checking the buffer usage state to decide to drop packets or not, I think atomic is better than semaphore.

Reply:
Agree